listener.go

```diff
@@ -43,6 +44,12 @@ func (h *apigeeSyncHandler) processSnapshot(snapshot *tran.Snapshot) {
 	for _, dbMan := range h.dbMans {
 		dbMan.SetDbVersion(snapshot.SnapshotInfo)
 	}
+	// retrieve encryption keys
+	orgs, err := h.dbMans[0].GetOrgs()
+	if err != nil {
+		log.Panicf("Failed to get orgs: %v", err)
+	}
+	h.cipherMan.AddOrgs(orgs)
 	// idempotent init api for all packages
```

Comment:
What about the scenario where a new Org is added as a dynamic scope?

Reply:
This is a `TODO` comment in `apigeeSyncHandler.Handle` line #60.
In Alpha rback, each cluster has only one org, and org cannot be changed dynamically, so this should be fine now.